### PR TITLE
pages*: sync https://developer.android.com more info links

### PR DESCRIPTION
**Pull request by vitorhcl**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

I forgot doing this on #12284, so I ran a `set-more-info-link.py -S`

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]

**Review by acuteenvy — COMMENTED** [review]
https://source.android.com/devices/tech/dalvik redirects to https://source.android.com/docs/core/runtime, not https://developer.android.com/tools/#art_and_dalvik.

The `#art_and_dalvik` part seems to have no effect, since there is no such section.

None of these pages provide information about the `dalvikvm` command.

File: pages.es/android/dumpsys.md
Comment (kant) [inline pages.es/android/dumpsys.md:1]: LGTM

File: pages.es/android/dalvikvm.md
Comment (kant) [inline pages.es/android/dalvikvm.md:1]: LGTM

File: pages.es/android/am.md
Comment (kant) [inline pages.es/android/am.md:1]: LGTM

File: pages.es/android/logcat.md
Comment (kant) [inline pages.es/android/logcat.md:1]: LGTM

File: pages.es/android/pm.md
Comment (kant) [inline pages.es/android/pm.md:1]: LGTM

File: pages.es/android/screencap.md
Comment (kant) [inline pages.es/android/screencap.md:1]: LGTM

File: pages.es/common/adb.md
Comment (kant) [inline pages.es/common/adb.md:1]: LGTM

File: pages.es/common/adb-reverse.md
Comment (kant) [inline pages.es/common/adb-reverse.md:1]: LGTM

File: pages.es/common/adb-install.md
Comment (kant) [inline pages.es/common/adb-install.md:1]: LGTM

**Review by kant — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Review by Magrid0 — APPROVED** [review]

**Comment by CleanMachine1** [comment]
Try avoid pinging that many people

**Comment by vitorhcl** [comment]
It is automatic

**Comment by kbdharun** [comment]
> Try avoid pinging that many people

Hi @CleanMachine1, glad to have you back :)

As Vitorhcl says this process of requesting reviews is automatic as we have adopted CODEOWNERs since last year. This allows native speakers (who are also collaborators) to be requested for a PR review automatically whenever the files in this language directories are modified. See https://github.com/tldr-pages/tldr/blob/main/COMMUNITY-ROLES.md#codeowners for more information.

Unfortunately, mass requesting review is something we can't eliminate atm when multiple translations are modified. We generally suggest maintainers avoid them (if they get merged soon). Even if we remove the review request now, they are already subscribed so we can't eliminate this option. IDK if there is a discussion/feature request already, I will try to raise this issue upstream to maybe add an option to bypass code owner's checks for collaborators if a lot of code owned files are modified.

**Review by isaacvicente — APPROVED** [review]

**Review by Magrid0 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**